package-lock.json

```diff
@@ -1989,6 +1989,21 @@
         "@types/yargs": "^13.0.0"
       }
     },
+    "@kwsites/file-exists": {
```

Comment:
This diff is more extensive than I'd like.

I just did:

```
npm i @wordpress/e2e-test-utils@latest
npm i @wordpress/env@latest
```

But it still seems that other packages got updated.